### PR TITLE
Data Challenge 1 modifications

### PR DESCRIPTION
**Pull request by smsimon**
Correction on rhombus gap in detector wafer layout

### CODE REVIEW

**Comment by smsimon** [comment]
Note: Did not change band conventions in any reference_tool folders since they will likely use the "Design tool simulations" release of the code

**Comment by smsimon** [comment]
Still awaiting updated noise and optics info for SAT MF changes.

File: AoA/scan_strategies/analyze_schedule.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    schedule = np.genfromtxt(fname, skip_header=3).T
+
+    arr = np.genfromtxt(
+        fname,
+        skip_header=3,
+        usecols=[4, 5, 7, 8, 9, 10],
+        dtype="f8,f8,S30,f8,f8,f8",
+        names=["starts", "stops", "names", "az_mins", "az_maxs", "elevations"],
+    )
```
Comment (tskisner) [inline AoA/scan_strategies/analyze_schedule.py:42]: This seems a lot nicer than the ground schedule reading in toast (https://github.com/hpc4cmb/toast/blob/51f896b1867ef13c9e8c1476e0c1cfae24ab6fac/src/toast/schedule.py#L271).  Perhaps eventually we could unify ground schedule I/O into the `GroundSchedule` class like the `SatelliteSchedule`.  Then we could read / write them any time we need.

**Review by tskisner — APPROVED** [review]
A prolific amount of scripting!  I noticed that some PNG images got added, not sure if that was intentional.  Also seems like you have a cleaner way of loading ground schedules that might be useful upstream in toast.  No show stoppers though, so merge when ready.

**Pull request merged**